.appveyor.yml

```diff
@@ -40,12 +40,16 @@ before_build:
     --sub-ref=%APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH%:has=/sync$
     --print
   - python --version
-  - pip install -r tools/cbse/requirements.txt --upgrade
+  - python -m pip install -r tools/cbse/requirements.txt --upgrade
 
 build_script:
   - cmd: cmake --version
   - cmd: call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvars32.bat"
+  # We don't need iteration; the for loop executes only once. But the function for resolving
+  # something in PATH (like 'which') is in the FOR command
+  - cmd: FOR %%p IN (python.exe) DO set firstPython=%%~$PATH:p
```

Comment:
Should we be looking for python3?

Reply:
There's not usually any "python3.exe" available on Windows. The problem we face is two dueling Python 3 installations